tests/CMakeLists.txt

```diff
@@ -88,11 +88,10 @@ include_directories(
   ${OPAE_SDK_SOURCE}/tests)
 
 add_library(commonlib SHARED common_test.h common_test.cpp)
-target_link_libraries(commonlib opae-c ${GTEST_BOTH_LIBRARIES} ${jsoncpp_LIBRARIES})
+target_link_libraries(commonlib opae-c ${GTEST_BOTH_LIBRARIES})
```

Comment:
Does commonlib need to link with json-c?

Reply:
Good call, I think so.